rfcs/0032-ckb-vm-version-selection/0032-ckb-vm-version-selection.md

```diff
@@ -1 +1,157 @@
-In review, see <https://github.com/nervosnetwork/rfcs/pull/238>, <https://github.com/nervosnetwork/rfcs/pull/236>, and <https://github.com/nervosnetwork/rfcs/pull/237>
+---
+Number: "0032"
+Category: Consensus (Hard Fork)
+Status: Draft
+Author: Ian Yang
+Organization: Nervos Foundation
+Created: 2021-04-26
+---
+
+# CKB VM Version Selection
+
+## Abstract
+
+This RFC proposes a mechanism to choose the CKB VM version after the hard fork. What's included in the new CKB VM version is beyond the scope of this document.
```

Comment:
Recommend adding a link to the VM version 1 RFC as an example of a VM version

Reply:
```suggestion
This RFC proposes a mechanism to choose the CKB VM version after the hard fork. This document will not cover what's included in the new CKB VM version, please refer to [rfc33] and [rfc34] for the changes.
```